### PR TITLE
Arc Doc

### PR DESCRIPTION
**Pull request by dzakhar**
Need to define whether I correctly added ARC EMSDP section into example specific readmes or not, It seems like platforms are placed alphabetically, If it is true, then **ARC** EM SDP should be the first in the list.

### CODE REVIEW

File: tensorflow/lite/micro/examples/person_detection_experimental/README.md [hunk 1]
```diff
@@ -7,12 +7,66 @@ This uses the experimental int8 quantized version of the person detection model.
...
+
+### Initial setup
+
+Follow the instructions on the [ARC EM SDP Initial Setup](/tensorflow/lite/micro/tools/make/targets/arc/README.md#ARC-EM-Software-Development-Platform-ARC-EM-SDP) to get and install all required tools for work with ARC EM SDP. 
+
+### Generate Example Project
+
+The example project for ARC EM SDP platform can be generated with the following command:
+    
+    make -f tensorflow/lite/micro/tools/make/Makefile TARGET=arc_emsdp TAGS=no_arc_mli generate_person_detection_int8_make_project
```
Comment (JaccovG) [inline tensorflow/lite/micro/examples/person_detection_experimental/README.md]: copy paste error: TAGS=no_arc_mli

File: tensorflow/lite/micro/examples/person_detection_experimental/README.md [hunk 2]
```diff
@@ -7,12 +7,66 @@ This uses the experimental int8 quantized version of the person detection model.
...
+
+### Build and Run Example 
+
+For more detailed information on building and running examples see the appropriate sections of general descriptions of the [ARC EM SDP usage with TFLM](/tensorflow/lite/micro/tools/make/targets/arc/README.md#ARC-EM-Software-Development-Platform-ARC-EM-SDP). In the directory with generated project you can also find a *README_ARC_EMSDP.md* file with instructions and options on building and running. Here we only briefly mention main steps which are typically enough to get it started. 
+
+1.	You need to [connect the board](/tensorflow/lite/micro/tools/make/targets/arc/README.md#connect-the-board) and open an serial connection.
+
+2.	Go to the generated example project director 
+    
+        cd tensorflow/lite/micro/tools/make/gen/arc_emsdp_arc/prj/person_detection/make`
```
Comment (JaccovG) [inline tensorflow/lite/micro/examples/person_detection_experimental/README.md]: person_detection_int8

File: tensorflow/lite/micro/tools/make/targets/arc/README.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+### Connect the Board
+
+1.	Make sure Boot switches of the board (S3) are configured in the next way:
+<center>
+
+| Switch #  |   Switch position  |
+| :-------: | :----------------: | 
+|     1     |       Low (0)      | 
+|     2     |       Low (0)      | 
+|     3     |       Low (0)      |
```
Comment (JaccovG) [inline tensorflow/lite/micro/tools/make/targets/arc/README.md]: is this correct?

**Review by JaccovG — CHANGES_REQUESTED** [review]

File: tensorflow/lite/micro/examples/person_detection_experimental/README.md (hunk 1 above)
Comment (dzakhar) [reply tensorflow/lite/micro/examples/person_detection_experimental/README.md]: Fixed it in next commit.

File: tensorflow/lite/micro/examples/person_detection_experimental/README.md (hunk 2 above)
Comment (dzakhar) [reply tensorflow/lite/micro/examples/person_detection_experimental/README.md]: Agree and fixed.

File: tensorflow/lite/micro/tools/make/targets/arc/README.md (hunk 3 above)
Comment (dzakhar) [reply tensorflow/lite/micro/tools/make/targets/arc/README.md]: Nope, 3rd switch must be in HIGH (1) state. Fixed it

**Review by JaccovG — APPROVED** [review]

**Pull request merged**